platform/linux-generic/odp_ipsec.c

```diff
@@ -834,11 +901,12 @@ static ipsec_sa_t *ipsec_in_single(odp_packet_t pkt,
 		if ((crypto.cipher_status.alg_err ==
 		     ODP_CRYPTO_ALG_ERR_ICV_CHECK) ||
 		    (crypto.auth_status.alg_err ==
-		     ODP_CRYPTO_ALG_ERR_ICV_CHECK))
+		     ODP_CRYPTO_ALG_ERR_ICV_CHECK)) {
+			ODP_ERR("ICV_CHECK failed\n");
```

Comment:
I do not think ODP_ERR() should be called here. In a sense, this is not an error (like e.g. some allocation failure) but normal operation.

Outputting an error log here would allow an attacker to fill the error log by sending us bogus or replayed IPsec packets. I think this should be at most ODP_DEBUG.

Reply:
ack, ODP_ERR() is not required, will remove it.